CRM/Activity/Form/ActivityLinks.php

```diff
@@ -30,7 +30,7 @@ public function buildQuickForm() {
   public static function commonBuildQuickForm($self) {
     $contactId = CRM_Utils_Request::retrieve('cid', 'Positive', $self);
     if (!$contactId) {
-      $contactId = CRM_Utils_Request::retrieve('cid', 'Positive', CRM_Core_DAO::$_nullObject, FALSE, NULL, $_REQUEST);
+      $contactId = CRM_Utils_Request::retrieve('cid', 'Positive', CRM_Core_DAO::$_nullObject, FALSE, NULL);
```

Comment:
It looks to me like the last 3 params here are all default, so this could be further simplified to 
```suggestion
      $contactId = CRM_Utils_Request::retrieve('cid', 'Positive');
```